core/src/main/java/hivemall/regression/RegressionBaseUDTF.java

```diff
@@ -123,10 +136,14 @@ public void process(Object[] args) throws HiveException {
         if(featureVector == null) {
             return;
         }
+        if (accDelta == null) {
+            accDelta = new FeatureValue[featureVector.length];
```

Comment:
`featureVector.length` can vary for each training instance.

So, `accDelta` should be `Set<FeatureValue>` or `Map<Object, Float>`.